source/develop/release-management/features/storage/incremental-backup.md

```diff
@@ -246,15 +260,11 @@ POST /vms/vm-uuid/backups
 ```
 
 Response:
-```
+```xml
 <backup id="backup-uuid">
-    <disks>
-        <disk id="disk-uuid" />
-        ...
-        ...
-    </disks>
+    <link href="/ovirt-engine/api/vms/vm-uuid/backups/backup-uuid/disks" rel="disks"/>
```

Comment:
Interesting that we never returned what was promised here.

We need to show explain that the user must do another request
to get the disks, and show the request:

    GET /ovirt-engine/api/vms/e007234a-7a20-4b32-9845-b724a7a7526a/backups/d73f52cf-0ff3-4392-a124-4d7e74fc8de7/disks

And the response:

    <disks>
        <disk id="60561da2-3776-49ff-90f3-f8bd54dbff30">
            <backup_mode>full</backup_mode>
            ...
        </disk>
        ...
    </disks>